### ISSUE TITLE
numberOfLines must be a non-negative number error

### ISSUE THREAD

**Issue by e-labInnovations** [mikezzb/react-native-timetable#1]
`'numberOfLines' in <Text> must be a non-negative number, received: -66. The value will be set to 0.`

```js
events = [
  {
    courseId: 'നൂൽപുട്ട്, വെള്ളക്കടല',
    day: 6,
    startTime: '19:00',
    endTime: '20:30'
  },
]
```

**Comment by binhhong1412** [comment]
I think because the Event card is too short (19:00 - 20:30) that it cannot show all the information. I tried extending the starTime/endTime and the bug is gone.

**Comment by mikezzb** [comment]
Thank you for your report! It shall be fixed in version 0.2.4.
